### PR TITLE
option to disable snake case transforms

### PR DESCRIPTION
**Pull request by julesjcraske**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const jsonClient = require('@cuvva/json-client');
```
Comment (nchagrass) [inline lib/index.js]: we need the dependency to be the `@cuvva` one for the request-id tracing

Comment (julesjcraske) [reply lib/index.js]: Ah merge conflict oops

**Review by mfiedorowicz — COMMENTED** [review]
Please add some tests 🙏🏻

**Review by nchagrass — APPROVED** [review]
I leave it to you to add test or not

File: lib/index.js [hunk 2]
```diff
@@ -7,9 +7,15 @@ function crpc(baseUrl, options) {
 	const client = jsonClient(baseUrl, options);
 
 	return function crpcRequest(path, body, options) {
-		const transformedBody = snek(body);
+		if (!options || !options.disableTransform) {
```
Comment (mfiedorowicz) [inline lib/index.js:10]: We could also use just:
```
if (!options?.disableTransform) {
    ...
}
```

Comment (julesjcraske) [reply lib/index.js:10]: Sourcery!

**Comment by mfiedorowicz** [comment]
Also, please bump version in the `package.json` and maybe mention this new option in the README.

**Comment by julesjcraske** [comment]
> Also, please bump version in the `package.json` and maybe mention this new option in the README.

I thought versions in package jsons were deprecated 10 years ago in favour of git tags? :P

All good points though.

**Comment by nchagrass** [comment]
> > Also, please bump version in the `package.json` and maybe mention this new option in the README.
> 
> I thought versions in package jsons were deprecated 10 years ago in favour of git tags? :P
> 
> All good points though.

tagging master once merged will package and publish the new version

**Comment by julesjcraske** [comment]
actually I think I can just use json-client

**Pull request closed without merging**